include/rnp/rnp2.h

```diff
@@ -260,6 +264,54 @@ rnp_result_t rnp_key_have_public(rnp_key_handle_t key, bool *result);
 
 /* TODO define functions for password-based encryption */
 
+rnp_result_t rnp_op_sign_create(rnp_op_sign_t *op,
+                                rnp_ffi_t      ffi,
+                                rnp_input_t    input,
+                                rnp_output_t   output);
+
+rnp_result_t rnp_op_sign_set_signing_key(rnp_op_sign_t op, rnp_key_handle_t key);
+
+rnp_result_t rnp_op_sign_set_compression(rnp_op_sign_t op, const char *compression, int level);
+rnp_result_t rnp_op_sign_set_armor(rnp_op_sign_t op, bool armored, bool cleansign);
+rnp_result_t rnp_op_sign_set_detached(rnp_op_sign_t op, bool detached);
+
+rnp_result_t rnp_op_sign_set_timestamps(rnp_op_sign_t op,
+                                        uint32_t      creation_time,
+                                        uint32_t      expiration_time);
```

Comment:
From these two only creation_time is timestamp, expiration time internally is counted in seconds. Or do you convert it from timestamp?

Reply:
That is just seconds until expiration, parameter name changed.